### ISSUE TITLE
Factsheet Button main page: link directly to scenario bundle factsheets, rework overview page 

### ISSUE THREAD

**Issue by han-f** [OpenEnergyPlatform/oeplatform#1418]
## Description of the issue

Could we link from the button of the main page directly to the scenario bundle factsheets? I think these will be the future heart of the factsheets.

I am not sure that we do need to factsheet overview page any longer. If we had short introductions on top of scenario bundles and model / framework factsheet pages, we could save some effort when updating as we then do not need to also think about the overview page. 

![grafik](https://github.com/OpenEnergyPlatform/oeplatform/assets/36844029/98b52ee0-8bcb-40b5-ba2a-131279bed77d)

## Ideas of solution

- Link from button to scenario bundle factsheet page
-  Add short (!) introductory text to top of:
    - scenario bundle factsheet page
    - model factsheet page
    - framework factsheet page
 - Remove "Overview" from "Factsheets" drop-down menu

## Workflow checklist
- [x ] I am aware of the workflow in [CONTRIBUTING.md](https://github.com/OpenEnergyPlatform/oeplatform/blob/develop/CONTRIBUTING.md)

as info cc @stap-m @l-emele @christian-rli

**Comment by jh-RLI** [comment]
It would also be great if we provided some introductory text that mentions the scenario comparison. Currently it seems that it is part of a scenario bundle, but it is more of a separate feature and should stand out a bit more.

**Comment by jh-RLI** [comment]
I chnaged the homepage and navigation bar and also changed the text a bit. 
![image](https://github.com/OpenEnergyPlatform/oeplatform/assets/38939526/054f19b6-5130-4999-bc14-d9b532a50d9d)

@han-f 
What do you think about the text?
It would be great if you could provide some text for the scenario bundle page. I removed the old overview page.

**Comment by han-f** [comment]
Thanks  @jh-RLI  for implementing this. I like the text and would only suggest slight adjustments as follows:
_Do you want to learn more about scenarios, the models used to project them and the actual data? Then, this is the right place for you. If you contributed data to the OEP this is the place where you can provide more context by creating your own scenario bundle._ 

What do you think?

I'll post an idea for the scenario bundle page intro text asap in a comment below.

**Comment by jh-RLI** [comment]
I like your update and will add it. 

FYI:
Today Adel and I will update the toep so we can have an oep release by the end of next week. I would like to include this change in this release. So we have time to find the optimal text also till end of next week. :)

PS
I also updated the navigation bar at the top. I noticed that we had several versions of the navigation bar on the home page, each with different content ... now they are harmonized.

**Comment by han-f** [comment]
Here is a suggestion for the intro to the scenario bundle page. I would like to suggest linking to the corresponding oeo classes in the textual viewer, as this is an indirect way of letting others make use of the ontology, without necessarily being aware of it. I think the textual viewer is really good and having a search function and back and forth navigation in it may even motivate others to navigate through the oeo and learn indirectly. 


[Scenario bundles](http://openenergy-platform.org/ontology/oeo/OEO_00020227) weave together important information about one or more [scenarios](http://openenergy-platform.org/ontology/oeo/OEO_00000364). They inform about [models](http://openenergy-platform.org/ontology/oeo/OEO_00020353) that were used to  project a scenario into the future (= [scenario projection](http://openenergy-platform.org/ontology/oeo/OEO_00010262)), and [studies](http://openenergy-platform.org/ontology/oeo/OEO_00020011) made based on a scenario, including publicatione (= [study report](http://openenergy-platform.org/ontology/oeo/OEO_00020012)). If there is quantitative [input data](http://openenergy-platform.org/ontology/oeo/OEO_00030029) and / or [output data](http://openenergy-platform.org/ontology/oeo/OEO_00020013) available on the OEP, the scenario bundles will link to that data, too. 

In a nutshell: A scenario bundle provides you with all relevant information to understand a scenario's context and to ease a potential re-use of quantitative data for your own purposes.




what do you think @stap-m @l-emele @wingechr ?

PS: exogenous  data does not have an alternate label "input data" while endogenous data does have an alternate label "output data". Should we add the label to the oeo? And should we maybe add an alternate label "publication" to study report?

**Comment by jh-RLI** [comment]
I like that already. In my opinion, the information about the model and framework factsheets is missing. The factsheet creation feature is separate from the scenario bundles page, but they can be selected when creating a bundle. This may not be clear to the user. I think we should at least mention them very briefly.

**Comment by han-f** [comment]
Thanks @jh-RLI 

How about this (minimally invasive so as not to move focus to factsheet pages). I could then also draft intro text for the model and framework factsheets separately. 

[Scenario bundles](http://openenergy-platform.org/ontology/oeo/OEO_00020227) weave together important information about one or more [scenarios](http://openenergy-platform.org/ontology/oeo/OEO_00000364). They inform about [models](http://openenergy-platform.org/ontology/oeo/OEO_00020353) (see also [model factsheets](https://openenergy-platform.org/factsheets/models/)) and frameworks (see [framework factsheets](https://openenergy-platform.org/factsheets/frameworks/)) that were used to  project a scenario into the future (= [scenario projection](http://openenergy-platform.org/ontology/oeo/OEO_00010262)) and [studies](http://openenergy-platform.org/ontology/oeo/OEO_00020011) made based on a scenario, including publications (= [study report](http://openenergy-platform.org/ontology/oeo/OEO_00020012)). If there is quantitative [input data](http://openenergy-platform.org/ontology/oeo/OEO_00030029) and / or [output data](http://openenergy-platform.org/ontology/oeo/OEO_00020013) available on the OEP, the scenario bundles will link to that data, too. 

In a nutshell: A scenario bundle provides you with all relevant information to understand a scenario's context and to ease a potential re-use of quantitative data for your own purposes.

**Comment by stap-m** [comment]
Thanks for the proposal text and the implementation.
I like the text, too. I am just wondering about the order. Since models and framework factsheet were least taken care of recently, and they also appear rather "late" within the bundles, I am not sure about mentioning them as first thing.

**Comment by han-f** [comment]
Good point @stap-m , how about this:?


[Scenario bundles](http://openenergy-platform.org/ontology/oeo/OEO_00020227) weave together important information about one or more [scenarios](http://openenergy-platform.org/ontology/oeo/OEO_00000364). They inform about [studies](http://openenergy-platform.org/ontology/oeo/OEO_00020011) made based on a scenario, including publications (= [study report](http://openenergy-platform.org/ontology/oeo/OEO_00020012)). If there is quantitative [input data](http://openenergy-platform.org/ontology/oeo/OEO_00030029) and / or [output data](http://openenergy-platform.org/ontology/oeo/OEO_00020013) available on the OEP, the scenario bundles will link to that data, too. They can also inform about [models](http://openenergy-platform.org/ontology/oeo/OEO_00020353) (if available as a [model factsheet](https://openenergy-platform.org/factsheets/models/)) and frameworks (if available as a [framework factsheet](https://openenergy-platform.org/factsheets/frameworks/)) that were used to project a scenario into the future (= [scenario projection](http://openenergy-platform.org/ontology/oeo/OEO_00010262)).  

In a nutshell: A scenario bundle provides you with all relevant information to understand a scenario's context and to ease a potential re-use of quantitative data for your own purposes.

**Comment by han-f** [comment]
Slight modification:

[Scenario bundles](http://openenergy-platform.org/ontology/oeo/OEO_00020227) weave together important information about one or more [scenarios](http://openenergy-platform.org/ontology/oeo/OEO_00000364). They inform about [studies](http://openenergy-platform.org/ontology/oeo/OEO_00020011) made based on a scenario, including publications (= [study report](http://openenergy-platform.org/ontology/oeo/OEO_00020012)). If there is quantitative [input data](http://openenergy-platform.org/ontology/oeo/OEO_00030029) and / or [output data](http://openenergy-platform.org/ontology/oeo/OEO_00020013) available on the OEP, the scenario bundles **can** link to that data, too. They can also inform about [models](http://openenergy-platform.org/ontology/oeo/OEO_00020353) (if available as a [model factsheet](https://openenergy-platform.org/factsheets/models/)) and frameworks (if available as a [framework factsheet](https://openenergy-platform.org/factsheets/frameworks/)) that were used to project a scenario into the future (= [scenario projection](http://openenergy-platform.org/ontology/oeo/OEO_00010262)).

In a nutshell: A scenario bundle provides you with all relevant information to understand a scenario's context and to ease a potential re-use of quantitative data for your own purposes.

**Comment by han-f** [comment]
@adelmemariani, @jh-RLI 
I just checked the toep, and while the text and button show up on the main page, the link when clicking the scenario bundle button goes to here: https://toep.iks.cs.ovgu.de/factsheets/overview/
which seems to be the non-existant overview page, and not the scenario bundle page, not sure if this is intentional on toep, thus flagging it here - do we have an extra place to comment on toep?. 
![grafik](https://github.com/OpenEnergyPlatform/oeplatform/assets/36844029/8fd7a029-3fa3-41ca-8ac9-fccff169eeca)
![grafik](https://github.com/OpenEnergyPlatform/oeplatform/assets/36844029/a3bb9786-8202-450b-8ffd-b851651b3048)

**Comment by jh-RLI** [comment]
Well spotted @han-f . I'll fix a few things now and ping Adel for another Toep update. This time I will also send an email. It was to be expected that some things are not "done" yet, but I definitely missed the outdated link :) 

We don't have a specific TOEP issue or anything like that yet.
In general, I think it's fine to comment on TOEP-related bugs in threads related to implementation (like this one). As long as it is stated that the bug was found in TOEP.

**Comment by jh-RLI** [comment]
> Slight modification, highlighted in bold, no need for bold in actual text:
> 
> [Scenario bundles](http://openenergy-platform.org/ontology/oeo/OEO_00020227) weave together important information about one or more [scenarios](http://openenergy-platform.org/ontology/oeo/OEO_00000364). They inform about [studies](http://openenergy-platform.org/ontology/oeo/OEO_00020011) made based on a scenario, including publications (= [study report](http://openenergy-platform.org/ontology/oeo/OEO_00020012)). If there is quantitative [input data](http://openenergy-platform.org/ontology/oeo/OEO_00030029) and / or [output data](http://openenergy-platform.org/ontology/oeo/OEO_00020013) available on the OEP, the scenario bundles **can** link to that data, too. They can also inform about [models](http://openenergy-platform.org/ontology/oeo/OEO_00020353) (if available as a [model factsheet](https://openenergy-platform.org/factsheets/models/)) and frameworks (if available as a [framework factsheet](https://openenergy-platform.org/factsheets/frameworks/)) that were used to project a scenario into the future (= [scenario projection](http://openenergy-platform.org/ontology/oeo/OEO_00010262)).
> 
> In a nutshell: A scenario bundle provides you with all relevant information to understand a scenario's context and to ease a potential re-use of quantitative data for your own purposes.

@bmlancien @adelmemariani can one of you add this text to the top of the scneario bundles overview page?
